code/game/machinery/engi_points_vendor

```diff
@@ -0,0 +1,35 @@
+/*	
+	var/static/list/prize_list = list(
+		new /datum/GBP_equipment("Tendie",				/obj/item/weapon/reagent_containers/food/snacks/nugget,								50,		1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,													50,		1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,																							50,		1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,																				250,	1),
+		new /datum/GBP_equipment("Advanced Indoor Fulton Pack",			/obj/item/weapon/extraction_pack/advanced,					300,	1),
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,											400,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,															500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,															500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,											500,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,																		600,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,															750,	3),
+		new /datum/GBP_equipment("Jetpack Upgrade x3",		/obj/item/weapon/tank/jetpack/suit,														1000,	3),
+		new /datum/GBP_equipment("Forcefield Projector x3",		/obj/item/device/forcefield,															1500,	3),
+		new /datum/GBP_equipment("Powertools x4",			/obj/item/weapon/storage/belt/utility/chief/full,									2000,	4),
+		new /datum/GBP_equipment("Freon Canister",			/obj/machinery/portable_atmospherics/canister/freon,						2500,	1),
+		new /datum/GBP_equipment("BZ Gas Canister",			/obj/machinery/portable_atmospherics/canister/bz,								2500,	1),
```

Comment:
did we kill the BZ sprite?

I figured it might work well in the new prototype can for extended release but I don't really care that much one way or another.